src/consensus.rs

```diff
@@ -486,149 +486,150 @@ impl Consensus {
             }
         }
 
+        // If this is the origin peer of the cluster, try to add origin peer to consensus
+        if let Err(err) = self.try_add_origin() {
```

Comment:
Why didn't we put `try_add_origin` in here in the first version? Is that related to refactoring, or just better?

Reply:
IIRC, when I added `try_add_origin`, I did not yet implement the "instantly self-elect itself as a leader" feature (right above `try_add_origin` call), so I had to put it inside the consensus loop.